### PR TITLE
#1433 feat add search function to the ideation portal

### PR DESCRIPTION
**Pull request by rz-p**

### CODE REVIEW

**Comment by reyhanrumengan** [comment]
In the Archive, when I search for an idea and if the Idea that were shown only one, the search bar were moved to the center and the IdeaCard were moved to the right

Bildschirmfoto 2023-06-20 um 10 36 00

**Review by reyhanrumengan — APPROVED** [review]

**Comment by LetsPoke** [comment]
Looks good 

is it possible to change the number of ideas to "searchIdea.length" when the searchTerm is not empty 
so that if 2 ideas are shown with seacht that the number of ideas is not saying 10

File: hms-app/src/pages/Archive.tsx [hunk 1]
```diff
@@ -1,24 +1,26 @@
 import React, { useEffect, useState, useContext } from 'react'
 import HackathonSelectDropdown from '../components/HackathonSelectDropdown'
-import { Group, Text, Checkbox, Title, Stack } from '@mantine/core'
+import { Group, Text, Checkbox, Title, Stack, Input } from '@mantine/core'
```
Comment (LetsPoke) [inline hms-app/src/pages/Archive.tsx:3]: unused import

File: hms-app/src/pages/Archive.tsx [hunk 2]
```diff
@@ -52,24 +54,28 @@ export default function Archive() {
     })
   }
 
-  const userIdea = relevantIdeaList.filter((item) => {
+  const handleChangeSearch = (event: React.ChangeEvent<HTMLInputElement>) => {
```
Comment (LetsPoke) [inline hms-app/src/pages/Archive.tsx:57]: unused const

**Review by LetsPoke — APPROVED** [review]

**Pull request merged**